core/authentication_api.php

```diff
@@ -717,6 +717,20 @@ function auth_does_password_match( $p_user_id, $p_test_password ) {
 	}
 
 	$t_password = user_get_field( $p_user_id, 'password' );
```

Comment:
I thought about it, but decided to leave it as-is because it's not necessarily a hash (think PLAIN or BASIC_AUTH login methods), and the underlying table column is called _password_